### PR TITLE
[#1510] Added global init function. 

### PR DESCRIPTION
**Pull request by rinkeshbhutwala**
Added new init function for setting global token values.

Signed-off-by: Rinkesh Bhutwala rbhutwal@proximabiz.com

### CODE REVIEW

File: cla-backend-go/main.go [hunk 1]
```diff
@@ -19,8 +25,25 @@ var (
...
 
+func init() {
+	stage := viper.GetString("STAGE")
+
+	awsSession, err := ini.GetAWSSession()
+	if err != nil {
+		log.Panicf("Unable to load AWS session - Error: %v", err)
+	}
+
+	configFile, err := config.LoadConfig(configFile, awsSession, stage)
```
Comment (dealako) [inline cla-backend-go/main.go:40]: I think now we are loading the config twice.  Can we be smart and cache the results the second time this is called?

**Review by dealako — CHANGES_REQUESTED** [review]
Let's discuss as I don't want to load the config twice.

File: cla-backend-go/main.go (hunk 1 above)
Comment (rinkeshbhutwala) [reply cla-backend-go/main.go]: Fixed loading config twice issue.

**Review by dealako — APPROVED** [review]
👍

**Pull request merged**